seabird.proto

```diff
@@ -5,82 +5,74 @@ option go_package = ".;pb";
 package seabird;
 
 // Identity is included in every message that requires auth. One AuthMethod at a
-// time is supported. For now, the only supported method is a token. You can
-// obtain a full Identity from the OpenSession method.
+// time is supported. For now, the only supported method is a token, obtained
+// from the server.
```

Comment:
`obtained from the server` implies that you need to make an RPC to get the auth token. Can you adjust the wording to make it clearer that the server maintainer needs to create the auth tokens separately, and that they then distribute them?

We also need to add this to the `README`.